src/synapse/workload.py

```diff
@@ -289,6 +289,24 @@ def enable_metrics(container: ops.Container) -> None:
         raise EnableMetricsError(str(exc)) from exc
 
 
+def disable_password_config(container: ops.Container) -> None:
```

Comment:
Sorry, but I have not seen where this function is called? Shouldn't it be called when the config is set to false?

Reply:
I have no idea what happened, maybe it was lost while merging with previous PRs, but now I added to change_config the call to this function.